package.json

```diff
@@ -43,26 +45,33 @@
   "devDependencies": {
     "broccoli-asset-rev": "^3.0.0",
     "chai": "^4.0.2",
-    "ember-cli": "~2.16.0",
+    "chai-things": "^0.2.0",
+    "ember-cli": "~3.4.3",
     "ember-cli-babel": "^7.1.2",
     "ember-cli-blueprint-test-helpers": "^0.19.1",
     "ember-cli-dependency-checker": "^3.0.0",
     "ember-cli-htmlbars": "^3.0.1",
-    "ember-cli-htmlbars-inline-precompile": "^1.0.0",
+    "ember-cli-htmlbars-inline-precompile": "^1.0.3",
     "ember-cli-inject-live-reload": "^2.0.0",
-    "ember-cli-qunit": "^4.0.1",
-    "ember-cli-shims": "^1.1.0",
-    "ember-cli-sri": "^2.1.0",
-    "ember-cli-uglify": "^2.0.0",
-    "ember-disable-prototype-extensions": "^1.1.0",
+    "ember-cli-qunit": "^4.3.2",
+    "ember-cli-sri": "^2.1.1",
+    "ember-cli-uglify": "^2.1.0",
+    "ember-disable-prototype-extensions": "^1.1.3",
     "ember-export-application-global": "^2.0.0",
-    "ember-load-initializers": "^1.0.0",
+    "ember-load-initializers": "^1.1.0",
+    "ember-maybe-import-regenerator": "^0.1.6",
     "ember-resolver": "^5.0.1",
     "ember-source": "~3.5.0",
+    "ember-source-channel-url": "^1.1.0",
+    "ember-try": "^1.0.0",
+    "es6-promise": "^4.0.0",
```

Comment:
do we need this dependency for anything?

Reply:
Will try removing.

Reply:
Removed.